### DISCUSSION TITLE
call endpoint from .ts files

### DISCUSSION THREAD

**Discussion by elenaaralla** [serenity-is/Serenity#7235] [Help] [answered]
Hi,

I'm using StartSharp 8 and I need to call an endpoint (to check the entity) from xyzDialog.ts file.

I'm new to fetch API, too.

Is there an example on demo app?
Or documentation?
Thank you,
Elena.

**Comment by L30P1N0** [comment]
`


        protected getService() { return yourEntityService.baseUrl; }


    yourEntityService.List({
        Take: 100,
        EqualityFilter: {
           [yourEntityRow.Fields.Id]: yourId
        },
    }, response => {
                  // do something
            }
        }


    }, { async: false })



`

**Comment by elenaaralla** [comment, marked as answer]
Thank you @L30P1N0!!!
